epam-promo/components/widgets/StatusIndicator.tsx

```diff
@@ -0,0 +1,17 @@
+import { withMods } from '@epam/uui-core';
+import { StatusIndicator as UuiStatusIndicator, StatusIndicatorProps as UuiStatusIndicatorProps } from '@epam/uui';
+import { EpamPrimaryColor } from '../types';
+
+export interface StatusIndicatorMods extends Omit<UuiStatusIndicatorProps, 'color'> {
+    color?: EpamPrimaryColor | 'white' | 'gray60' | 'yellow'| 'orange' | 'fuchsia' | 'purple' | 'violet' | 'cobalt' | 'cyan' | 'mint';
```

Comment:
```suggestion
    color?: EpamPrimaryColor | 'white' | 'gray' | 'yellow'| 'orange' | 'fuchsia' | 'purple' | 'violet' | 'cobalt' | 'cyan' | 'mint';
```
use "gray" to avoid future deprecational

Reply:
fixed